part-4/model-building-components.md

```diff
@@ -564,15 +566,111 @@ The screen should look like in the screen shot below:
+{% hint style="info" %}
+In contrast to other model structures with container criteria, an empty criteria for the events group means that the group will **not** be created in any container! The typical criteria for the events group is the `Events` node that is automatically created in an empty spatial structure.
+{% endhint %}
+
+### Application
+
+Application is a special type of an event group that is defined for a specific **molecule**.‌ In addition to the properties of an event group, an application has specify an **administered molecule** and the path to the **Application Molecule Builder**.
+
+ - **Application Model Builder**: Applications that add molecules to the system require an application model builder that is a virtual compartment for the administered molecule.
+
+An Application can be created within:
+    - an event group,
+    - another application.
+
+### Event
 
-In the drug delivery case, however, you rather want to use an application instead of an event to have more options available for drug release and repeated applications. Since the generation of an application in MoBi® can be rather complicated and is beyond the scope of this manual, we will restrict the description to adapting applications that were previously imported from PK-Sim®, where complex applications schemes can be generated more easily.
+An **Event** is the actual event that changes something in the model, like the amount of a molecule or a parameter value.‌
 
-Events and applications are grouped in an events building block, where events are sub-grouped in event groups. To create such a structure, you may need to create a new events building block using the context menu of the building block explorer or the corresponding ribbon button of the Modeling & Simulation tab. In our example, an empty events building block named "Events" has already been created automatically. Simulations imported from a PK-Sim® project also contain such a building block that contains the applications.
+Events can be created within:
+    - an event (sub-)group,
+    - an application.
 
-To add events or applications to the project, you need to open the events building block for editing. This can be done by double-clicking on it or by using the **Edit** command of the context menu in the building block explorer.
+Each event has 
+    - parameters,
+    - a start condition,
+    - **One time** property - should the event be executed (maximal) once during the simulation? If true, the event will be executed the first time the condition is met. If false, the event will be executed each time the condition is met.
```

Comment:
_If true, the event will be executed **only** the first time when the condition is met._

Reply:
done